instance/github.py

```diff
@@ -162,3 +162,10 @@ def extra_settings(self):
         Extra settings contained in the PR body
         """
         return get_settings_from_pr_body(self.body)
+
+    @property
+    def github_pr_url(self):
+        """
+        Construct the URL for the pull request
+        """
+        return 'https://github.com/{fork_name}/pull/{number}'.format(fork_name=self.fork_name, number=self.number)
```

Comment:
@Kelketek I pushed this to production, and this actually still seem wrong - the migrations do the right thing, so this was hiding the issue when I tested locally, but this doesnt, cf https://github.com/open-craft/opencraft/blob/34782a79d1b4a08b0d59f93ef91ff7611c0ffb7e/instance/github.py#L97 - `fork_name` is again the repo from which the PR is sent, not the destination repo of the PR.